github/table_github_organization_dependabot_alert.go

```diff
@@ -0,0 +1,299 @@
+package github
+
+import (
+	"context"
+
+	"github.com/google/go-github/v48/github"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func gitHubDependabotAlertColumns() []*plugin.Column {
+	return []*plugin.Column{
+		{
+			Name:        "dependabot_number",
+			Type:        proto.ColumnType_INT,
+			Description: "The security alert number.",
+			Transform:   transform.FromField("Number"),
+		},
+		{
+			Name:        "state",
+			Type:        proto.ColumnType_STRING,
+			Description: "The state of the Dependabot alert.",
+		},
+		{
+			Name:        "dependency_package_ecosystem",
+			Type:        proto.ColumnType_STRING,
+			Description: "The package's language or package management ecosystem.",
+			Transform:   transform.FromField("Dependency.Package.Ecosystem"),
+		},
+		{
+			Name:        "dependency_package_name",
+			Type:        proto.ColumnType_STRING,
+			Description: "The unique package name within its ecosystem.",
+			Transform:   transform.FromField("Dependency.Package.Name"),
+		},
+		{
+			Name:        "dependency_manifest_path",
+			Type:        proto.ColumnType_STRING,
+			Description: "The unique package name within its ecosystem.",
```

Comment:
Could we update this description to `The unique manifestation path within the ecosystem.`?